bucket/winssh-pageant.json

```diff
@@ -0,0 +1,41 @@
+{
+    "version": "1.2",
+    "description": "Pageant to Windows OpenSSH agent bridge",
+    "homepage": "https://github.com/ndbeals/winssh-pageant",
+    "license": "BSD-3-Clause",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/ndbeals/winssh-pageant/releases/download/v1.2/winssh-pageant-v1.2_amd64.zip",
+            "hash": "a9bef9d134e872e60fd1d0950ce29f73b98af4802c9e93987a4ecafe8debf814"
+        },
+        "32bit": {
+            "url": "https://github.com/ndbeals/winssh-pageant/releases/download/v1.2/winssh-pageant-v1.2_386.zip",
+            "hash": "0591f0c4475dba8fbfacc7ffe5fcb77ac384008d884976b386b4f25fa894e3a8"
+        }
+    },
+    "bin": "winssh-pageant.exe",
+    "notes": [
+        "This application is meant to run in the background upon system startup. Do that by running the following commands:",
+        "$Shortcut = (New-Object -comObject WScript.Shell).CreateShortcut($env:APPDATA+'\\Microsoft\\Windows\\Start Menu\\Programs\\Startup\\winssh-pageant.lnk')","$Shortcut.TargetPath = '$dir\\winssh-pageant.exe'",
+        "$Shortcut.Save()"
```

Comment:
Use only verbal instructions to copy the shortcut file. This command will never be executed by sanity people.

Reply:
I'm doing the exact same thing https://github.com/lukesampson/scoop-extras/blob/master/bucket/ssh-agent-wsl.json is doing, echoing a command that users can run (or add to an autoran script, same thing). Why is that utility allowed to do it?

Reply:
it is different to run simple command with 2 parameters instead of some advanced powershell specific command. Lots of users are using cmd wrapper to run scoop.

Add notes about manually copying the shortcut file.

Reply:
Lots of other manifests (filezilla_server, grepwin, eagleget_portable, smarttaskbar) have notes telling users to run scripts in `$dir` (sometimes even as administrator). What I'm doing by outputting a 3 line command is no different. I'd say it's even more transparent than running a file.

Reply:
I'll just include a .ps1 script that does it then, just like filezilla_server is doing.

Reply:
It is different...

There is huge different between powershell and cmd. You cannot suggest running powershell commands to cmd users.

Reply:
But it's fine, I'll add notes about manual shortcuts.

Reply:
I don't think that's true that there is a huge difference (PowerShell has shipped with the OS for over a decade now).

But still, what about advising the user to use schtasks.exe instead to set the logon action?

Reply:
> I don't think that's true that there is a huge difference (PowerShell has shipped with the OS for over a decade now).

Not to mention the fact that the _official install instructions for scoop itself_ has the user downloading and executing a powershell script straight from the internet. (A La `curl <url> | sh` infamy). I realized I must have got Ash258 on a bad day or something, because half of the things he didn't want in this manifest exist in numerous other manifests in this bucket.


> But still, what about advising the user to use schtasks.exe instead to set the logon action?

Ultimately I don't really care how scoop tells users to auto-start this tool, that could do just fine.

I'm not going to spend any more time on this, the inconsistent standards in this repo has seen to that. If you want to use scoop to install this, you can find a manifest here: https://gist.github.com/ndbeals/08fd5f1eed011bce2e7bba06da0e5314